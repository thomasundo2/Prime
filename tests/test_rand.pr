int main()
{
   lint l1;
   lint l2;
   lint l3;
   l1 = 1l;
   l2 = 100l;
   l3 = 1000000000000000000000000000000000000000000l;
   printl(random(l1,l2));
   printl(random(1l,l3));
   printl(random(1l,10000l));
   return 0;
}
