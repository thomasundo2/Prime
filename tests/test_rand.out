59
386228593951045501005359438524419323443515
6098
